src/pages/UpSertUser/hooks/useGetUserByID.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from "graphql-request";
+import { useQuery } from "react-query";
+import { GetUserByIdQueryQuery, GetUserByIdQueryQueryVariables } from "src/generated/graphql";
+
+import useQueryClient from "src/hooks/useQueryClient";
+
+const useGetUserByID = (userId: string) => {
+    const queryClient = useQueryClient();
+    const result = useQuery(["GetUserByIDQuery", { _eq: userId }], async () => {
```

Comment:
```suggestion
    const result = useQuery(["GetUserByIDQuery", userId ], async () => {
```
{ _eq: userId } will always be a new object

Reply:
![image](https://user-images.githubusercontent.com/58205909/132112580-e309c231-b387-4b72-90da-e5d56773cfbf.png)